### PR TITLE
metrics: add alert for slow store

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

### What problem does this PR solve?

Add alerting for slow stores.

### What is changed and how it works?

Add expression to grafana and add alerting rule to alertmannager.
image

### Check List

Tests

- Unit test
- Integration test

Related changes

- Need to cherry-pick to the release branch

### Release note

```release-note
None
```

### CODE REVIEW

File: metrics/grafana/pd.json [hunk 1]
```diff
@@ -1352,101 +1360,95 @@
           "type": "table"
         },
         {
-          "cacheTimeout":null,
-          "colorBackground":false,
-          "colorValue":false,
-          "colors":[
+          "cacheTimeout": null,
```
Comment (lhy1024) [inline metrics/grafana/pd.json:1363]: why so much diff?

File: metrics/grafana/pd.json [hunk 2]
```diff
@@ -10833,4 +10835,4 @@
   "title": "Test-Cluster-PD",
   "uid": "Q6RuHYIWk",
   "version": 1
-}
+}
```
Comment (lhy1024) [inline metrics/grafana/pd.json:10838]: end line

File: metrics/grafana/pd.json (hunk 1 above)
Comment (5kbpers) [reply metrics/grafana/pd.json:1363]: It was format by my code editor.

File: metrics/alertmanager/pd.rules.yml [hunk 3]
```diff
@@ -168,3 +168,15 @@ groups:
...
       value: '{{ $value }}'
       summary: PD server has been restarted
+
+  - alert: PD_cluster_slow_tikv_nums
+    expr: (sum ( pd_cluster_status{type="store_slow_count"} ) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)
+    for: 1m
+    labels:
+      env: ENV_LABELS_ENV
+      level: critical
+      expr:  (sum ( pd_cluster_status{type="store_slow_count"} ) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)
```
Comment (rleungx) [inline metrics/alertmanager/pd.rules.yml:178]:
```suggestion
      expr:  (sum(pd_cluster_status{type="store_slow_count"}) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)
```

File: metrics/alertmanager/pd.rules.yml [hunk 4]
```diff
@@ -168,3 +168,15 @@ groups:
       description: 'cluster: ENV_LABELS_ENV, instance: {{ $labels.instance }}, values:{{ $value }}'
       value: '{{ $value }}'
       summary: PD server has been restarted
+
+  - alert: PD_cluster_slow_tikv_nums
+    expr: (sum ( pd_cluster_status{type="store_slow_count"} ) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)
```
Comment (rleungx) [inline metrics/alertmanager/pd.rules.yml:173]:
```suggestion
    expr: (sum(pd_cluster_status{type="store_slow_count"}) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)
```

File: metrics/alertmanager/pd.rules.yml [hunk 5]
```diff
@@ -168,3 +168,15 @@ groups:
...
+    for: 1m
+    labels:
+      env: ENV_LABELS_ENV
+      level: critical
```
Comment (rleungx) [inline metrics/alertmanager/pd.rules.yml:177]: critical or warning?

Comment (5kbpers) [reply metrics/alertmanager/pd.rules.yml:177]: I think it could be critical

**Review by rleungx — APPROVED** [review]

**Review by lhy1024 — APPROVED** [review]

**Pull request merged**